components/experimental/src/measure/measureunit.rs

```diff
@@ -34,9 +32,10 @@ pub struct MeasureUnit {
 }
 
 impl MeasureUnit {
-    /// Returns a reference to the single units contained within this measure unit.
-    pub fn get_single_units(&self) -> &SmallVec<[SingleUnit; 8]> {
-        &self.single_units
+    /// Returns a slice of references to the single units contained within this measure unit.
+    #[cfg(feature = "alloc")]
+    pub fn get_single_units(&self) -> &[SingleUnit] {
```

Comment:
nit: rust getters are not usually prefixed with `get_`
```suggestion
    pub fn single_units(&self) -> &[SingleUnit] {
```

Reply:
done